README.md

```diff
@@ -32,8 +30,8 @@ This project mainly tested on Linux, BSD, Windows, MAC and Android(experimental)
 # Roadmap for release
 - [x] Distributed Tree/Graph structure with automatic lookup
 - [x] Event path calculation between Tree/Graph nodes
-- [x] API Client subscriptions for each node and Event delivery
-- [x] Event broadcasting and round rubin load balancing using statefull path calculation
+- [x] API client subscriptions for each node and event delivery
+- [x] Event broadcasting and round rubin load balancing using stateful path calculation
```

Comment:
round robin?

Reply:
@emintham you are correct it should be `round robin`